master/service.lua

```diff
@@ -41,3 +52,7 @@ for _, framework in ipairs(state["frameworks"]) do
     end
   end
 end
+
+ngx.status = ngx.HTTP_NOT_FOUND
+ngx.say("404 Not Found: service " .. ngx.var.serviceid .. " unknown.")
```

Comment:
Can we add timestamp when was the cache refreshed? Related to https://mesosphere.atlassian.net/browse/DCOS-13596